qa/README.md

```diff
@@ -0,0 +1,46 @@
+**Application Test suite**
+--------------------------
+**Requirements**
+
+- Install Python 2.7
+- Donwload Sidechain SDK
+
+**Additional settings**
+
+Setup these environment variables:
+```
+BITCOINCLI: path to zen-cli
+BITCOIND: path to zend
+APP_JAR: sidechain app jar
+APP_LIB: sidechain app lib folder
+APP_MAIN: main sidechain class to run
+SIDECHAIN_SDK: path to SDK folder
```

Comment:
Seems, that we need to extract STF and Bootstaping tool to the separate git repo. Keep them generalized and let to set them as a submodules. So no need to copy that sources or keep SDK. Moreover it will let developers to fork these modules and change depends on their needs.
What do you think about this?

Reply:
I agree could be good to separate them from the SDK , since both  the SDK and customer apps will use them for testing.
do you know how submodules loading works in python ? maybe we need anyway to set an env variable to the module

Reply:
I think, that we can use a git submodule, so the code folder will be placed in the place specified by the sidechain developer for his specific project.

Reply:
yes, could be ok, in this way we can use relative path in python import